gaia-header.js

```diff
@@ -643,6 +650,26 @@ module.exports = component.register('gaia-header', {
         if (value) { this.setAttr('no-font-fit', ''); }
         else { this.removeAttr('no-font-fit'); }
       }
+    },
+
+    // The [ignore-dir] attribute can be used to force the older behavior of
+    // gaia-header, where the whole header is always displayed in LTR mode.
+    ignoreDir: {
+      get: function() {
+        if ('ignoreDir' in this) { return this._ignoreDir || false; }
+        return this.hasAttribute('ignore-dir');
+      },
+      set: function(value) {
+        debug('set ignore-dir', value);
+        value = !!(value || value === '');
+
+        if (value === this.ignoreDir) { return; }
+        this._ignoreDir = value;
+        this.dirObserver &= !value;
```

Comment:
If `value` is true, then we ignore the `dir` attribute and don’t need to observe it any more. So:
* if `dirObserver` is false, it should remain false
* if `dirObserver` is true but the `dir` must be ignored, set `dirObserver` to false